src/math/nfft/dnfft_1d_kernels.cu

```diff
@@ -97,7 +97,7 @@ void accumulateOnDevice(const double* M, const int ldm, const int sign, ScalarTy
                         const ScalarType* cubic_coeff, const int size, cudaStream_t stream_) {
   const auto& helper = HelperSelector<ScalarType>::value;
   const static int convolution_size = 2 * helper.get_oversampling() + 1;
-  const auto blocks = getBlockSize(size * size * convolution_size, 128);
+  const auto blocks = getBlockSize(size * size * convolution_size, 85);
```

Comment:
Could you add a comment to say where the magic number of 85 comes from

Reply:
Sorry, I was playing around with different multiples of conviultion_size (17 * 5 = 85), and forgot it there. There was not any measurable benefit.
Reverted it to 128 in 82b6ec5fe537316af981c1df0d884e514a0d68f2